package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "spectacle-monorepo",
   "scripts": {
-    "start:js": "pnpm run --filter spectacle-example-js start",
-    "start:ts": "pnpm run --filter spectacle-example-ts start",
-    "start:md": "pnpm run --filter spectacle-example-md start",
+    "start:js": "pnpm run build:core && pnpm run --filter spectacle-example-js start",
+    "start:ts": "pnpm run build:core && pnpm run --filter spectacle-example-ts start",
+    "start:md": "pnpm run build:core && pnpm run --filter spectacle-example-md start",
```

Comment:
should we watch these that way changes to the library update in the demo apps?

Reply:
Yeah, I think that's the right move. I had that same vision, but just didn't get it squeezed into this PR. Happy to try to do that, or we can tackle as a follow-up.

Reply:
To have a watch be a task doesn't it need to use some concurrent helper otherwise since the process never sees an exit it will never execute the right hand of the &&?

Reply:
Yeah, I think we'll want to pull in concurrently for this. Roemer setup a similar script in Victory using concurrently, we can borrow from that!